mobify/platform-scaffold
# fix low res images in cart and payment step

Cart thumbnail images are very poor quality

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1273

## Changes
- use the higher res images (same as plp) on cart and payment 

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to cart 
- go to cart and payment step to check the image
